### ISSUE TITLE
Expose segment reload status as JMX Metric

### ISSUE THREAD

**Issue by daniellavoie** [apache/incubator-pinot#6520]
It would be very helpful for operation purpose to have the ability to get quick overview on the reload status of a table. 

All I could find for now is the `GET /segments/{tableName}_OFFLINE/metadata?type=OFFLINE`. As I'm exploring its output, I realize that the reload status is specific to each individual segments. While providing a startree status there would be a good step forward, a global status would greatly improve the operation experience.

### Output from metadata endpoint

```
"myTable_OFFLINE_1602111600000_1602111600000_2670" : {
    "segmentName" : "myTable_OFFLINE_1602111600000_1602111600000_2670",
    ...
    "columns" : [ ],
    "indexes" : {
      "column1" : {
        "bloom-filter" : "NO",
        "dictionary" : "YES",
        "forward-index" : "YES",
        "inverted-index" : "NO",
        "null-value-vector-reader" : "NO",
        "range-index" : "NO"
      },
      "column2" : {
        "bloom-filter" : "NO",
        "dictionary" : "YES",
        "forward-index" : "YES",
        "inverted-index" : "NO",
        "null-value-vector-reader" : "NO",
        "range-index" : "NO"
      },
    }
  }
```

## Exposing reload status as JMX Metrics

In addition to a global status, exposing the individual reload status as well as the global status would be very helpful in regards to configuration alerts on a prometheus stack.

**Comment by fx19880617** [comment]
I feel it might be too much overhead to expose segment level status.
But possible to provide aggregated metrics at table level and maybe table/column level.

**Comment by daniellavoie** [comment]
Yes aggregated would be fine. Mostly, this metric is to answer `Is it down yet?`

**Comment by fx19880617** [comment]
Got it, then I  think we can add a gauge value in pinot server per table/column/index.
